Libraries/Components/Keyboard/KeyboardAvoidingView.js

```diff
@@ -30,24 +30,25 @@ type Props = $ReadOnly<{|
   ...ViewProps,
 
   /**
-   * Specify how to react to the presence of the keyboard.
+    Specify how to react to the presence of the keyboard.
+
+    > Android and iOS both interact with this prop differently. On both iOS and Android, setting `behavior` is recommended.
    */
   behavior?: ?('height' | 'position' | 'padding'),
 
   /**
-   * Style of the content container when `behavior` is 'position'.
+    The style of the content container(View) when behavior is 'position'.
    */
   contentContainerStyle?: ?ViewStyleProp,
 
   /**
-   * Controls whether this `KeyboardAvoidingView` instance should take effect.
-   * This is useful when more than one is on the screen. Defaults to true.
+    Enabled or disabled KeyboardAvoidingView. The default is `true`.
```

Comment:
The default value is given in the description in the documentation(the source of truth). I would suggest removing it from the description and adding a `@default` annotation for default value to keep it consistent. Or we can keep both and later remove it.

Reply:
I agree, `@default` is better here.